vxquery-core/src/main/java/org/apache/vxquery/xmlparser/SAXContentHandler.java

```diff
@@ -66,6 +66,7 @@
     protected final List<ElementNodeBuilder> enbStack;
     protected final List<ElementNodeBuilder> freeENBList;
     protected boolean isIndexHandler;
+    protected boolean index;
```

Comment:
If we really need to do some indexing specific things durning XML parsing, it would be nice to have an interface that it being called by the parser to implement the functionality (with a dummy or no implementation of the interface in case no indexing is desired.